pages/mentors.tsx

```diff
@@ -30,7 +30,10 @@ const App = () => {
   const isSmall = useMediaQuery("(max-width: 800px)");
 
   const [isListView, setIsListView] = useState(false);
-  const WAVES = [{ waveId: 4, waveName: "VJC Mentorship 2023" }];
+  const WAVES = [
+    { waveId: 4, waveName: "VJC Mentorship 2023" },
```

Comment:
Could this be placed below L35? So that "Wave 2023" (which is more recent) is showed first.